impala/hiveserver2.py

```diff
@@ -84,7 +84,8 @@ def rollback(self):
         raise NotSupportedError
 
     def cursor(self, user=None, configuration=None, convert_types=True,
-               dictify=False, fetch_error=True, close_finished_queries=True):
+               convert_strings_to_unicode=False, dictify=False, 
```

Comment:
To guarantee backward compatibility the new argument should be the last in the list + it should be True by default.

Reply:
Fixed